### PR TITLE
API 2563 compound id support

### PR DESCRIPTION
**Pull request by bschofield-va**
- Value mapping now supports composite columns
- Value mapping now supports composite columns

### CODE REVIEW

**Comment by bschofield-va** [comment]
FYI @department-of-veterans-affairs/lighthouse-pteracuda

**Review by jhulbert-va — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -119,41 +126,62 @@ For the JPA field value of `Vulcan`, here's how the following parameters match.
...
 
 For a table with the following values:
 
-| ID | FOOD   |
-| -- | ------ |
-| 1  | Nachos |
-| 2  | Tacos  |
-| 3  | A Shoe |
-| 4  | Nachos |
+| ID | FOOD | | -- | ------ | | 1 | Nachos | | 2 | Tacos | | 3 | A Shoe | | 4 | Nachos |
```
Comment (eclendenning-va) [inline README.md:137]: Looks like the formatter didnt like this table.

Comment (bschofield-va) [reply README.md]: good eye

**Review by eclendenning-va — APPROVED** [review]

**Pull request merged**